docs/source/contributing.rst

```diff
@@ -0,0 +1,35 @@
+Contributing to PGMax
+=====================
+
+PGMax is an open-source library, and as such, we very much value
+contributions from the community!
+
+Contributing code using pull requests
+-------------------------------------
+
+We welcome pull requests, in particular for documentation upgrades,
+additional examples, or those issues marked with `good first
+issue <https://github.com/vicariousinc/PGMax/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22>`__
+or
+`contributions-welcome <https://github.com/vicariousinc/PGMax/labels/contributions-welcome>`__.
+
+For other proposals, we ask that you first open a GitHub
+`Issue <https://github.com/vicariousinc/PGMax/issues/new/choose>`__  to seek
+feedback on your planned contribution.
+
+We do all of our development using git, so basic knowledge is assumed.
+
+Follow these steps to contribute code:
+
+1. Follow the `Developer Installation
+   Instructions <installation.html#developer>`__ to install a fork of PGMax
+   locally on a new branch.
+2. Develop and implement your changes with your favorite code editor (we
+   recommend `VS Code <https://code.visualstudio.com/>`__)
```

Comment:
Is this necessary :joy:

Reply:
not really - but they mentioned it in JAX's contribution guide, so I thought I'd do the same lol

Reply:
It stood out to me mainly because I use neovim instead of vs code so at least for me the recommendation is not sincere.... But fine to keep!